atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -243,20 +245,22 @@ def set_group_manifest_info(self, extra, worker_metadatas):
             repo = ImageName.parse(repositories[0]).to_str(registry=False, tag=False)
             # group_manifests added the registry, so this should be valid
             registries = self.workflow.push_conf.all_registries
+
+            digest_version = get_manifest_media_version(manifest_digest)
+            digest = manifest_digest.default[digest_version]
```

Comment:
Does this actually work?

```python
>>> md = ManifestDigest({'v1': 'digest'})
>>> ver = get_manifest_media_version(md)
>>> ver
'v1'
>>> md.default[ver]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: string indices must be integers
```

Reply:
```suggestion
            digest = manifest_digest.default
```

Test is wrong, see further down.

Reply:
Fixed.